waspc/src/Wasp/Analyzer/Evaluator/EvaluationError.hs

```diff
@@ -53,6 +53,8 @@ data EvalErrorCtx
 data EvaluationParseError
   = -- | In case when evaluation includes parsing with Parsec and it fails.
     EvaluationParseErrorParsec Text.Parsec.ParseError
+  | -- | In case when evaluation includes parsing with Megaparsec and it fails.
+    EvaluationParseErrorMegaparsec String
```

Comment:
Errors are already [prettified by Megaparsec](https://markkarpov.com/tutorial/megaparsec.html#displaying-parse-errors) by the time they reach here, as otherwise we'd be having to pass around the internal Psl `Parser` definition.